tests/platform_tests/api/test_sfp.py

```diff
@@ -262,7 +354,6 @@ def test_get_transceiver_info(self, duthosts, enum_rand_one_per_hwsku_hostname,
             if self.expect(info_dict is not None, "Unable to retrieve transceiver {} info".format(i)):
                 if self.expect(isinstance(info_dict, dict), "Transceiver {} info appears incorrect".format(i)):
                     actual_keys = info_dict.keys()
-                    
```

Comment:
Please revert this

Reply:
I have reverted already and committed.

Reply:
This is the fix to address the addtional line created in previous commit